jsonPath

```diff
@@ -1,6 +1,6 @@
-../../../inputJsonFiles/Architecture/jsonArch3PC2Workers1SourceMonitored.json
```

Comment:
Do we still need it after the change of python we did?

Reply:
Removed.